atmo/stats/models.py

```diff
@@ -0,0 +1,55 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, you can obtain one at http://mozilla.org/MPL/2.0/.
+from django.contrib.postgres.fields import JSONField
+from django.core.serializers.json import DjangoJSONEncoder
+from django.db import models
+from django.utils import timezone
+
+
+class Metric(models.Model):
+    created_at = models.DateTimeField(
+        editable=False,
+        blank=True,
+        default=timezone.now,
+        db_index=True,
+    )
+    key = models.CharField(
+        max_length=100,
+        db_index=True,
+        help_text='Name of the metric being recorded',
+    )
```

Comment:
I think the plan is to expose this in redash and we'll likely be ordering by `created_at` and grouping by the `key`. So I figured both should be indexed. I'll check out that link tho, thanks.